From rpjday@crashcourse.ca  Mon Nov 26 22:01:38 2007
From: "Robert P. J. Day" <rpjday@crashcourse.ca>
Date: Wed, 14 Nov 2007 16:21:00 -0500 (EST)
Subject: USB: linux/usb/Kbuild needs only "header-y", not "unifdef-y"
To: <linux-usb@vger.kernel.org>
Message-ID: <alpine.LFD.0.9999.0711141616450.13206@localhost.localdomain>


Given that none of the referenced header files test the proprocessor
conditional __KERNEL__, there's no point "unifdef"fing them.

Signed-off-by: Robert P. J. Day <rpjday@crashcourse.ca>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/usb/Kbuild |   10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

--- a/include/linux/usb/Kbuild
+++ b/include/linux/usb/Kbuild
@@ -1,7 +1,7 @@
-unifdef-y += audio.h
-unifdef-y += cdc.h
-unifdef-y += ch9.h
-unifdef-y += gadgetfs.h
-unifdef-y += midi.h
+header-y += audio.h
+header-y += cdc.h
+header-y += ch9.h
+header-y += gadgetfs.h
+header-y += midi.h
 unifdef-y += g_printer.h
 
